packages/contracts/contracts/votings/majority-voting/ERC20Voting.sol

```diff
@@ -0,0 +1,157 @@
+/*
+ * SPDX-License-Identifier:    MIT
+ */
+
+pragma solidity 0.8.10;
+
+import "@openzeppelin/contracts-upgradeable/token/ERC20/extensions/ERC20VotesUpgradeable.sol";
+
+import "./MajorityVoting.sol";
+
+contract ERC20Voting is MajorityVoting {
+
+    ERC20VotesUpgradeable public token;
+
+    mapping(uint256 => uint64) snapshotBlocks;
+
+    function getSnapshotBlock(uint256 _voteId) public view returns (uint64) {
+        return snapshotBlocks[_voteId];
+    }
+
+    /// @dev describes the version and contract for GSN compatibility.
+    function versionRecipient() external view virtual override returns (string memory) {
+        return "0.0.1+opengsn.recipient.ERC20Voting";
+    }
+
+    /// @dev Used for UUPS upgradability pattern
+    /// @param _dao The DAO contract of the current DAO
+    function initialize(
```

Comment:
I would have this method on the top cause it is the constructor. Also we would loose some consistency over the project.

Reply:
Changed order accordingly